### ISSUE TITLE
User is able to upload the large size Profile image. 

### ISSUE THREAD

**Issue by drasticdpk** [appirio-tech/connect-app#3772]
### Expected behavior
The  user should not be allowed to upload the large  size image of more than 2 MB like we have in Community app. .

### Actual behavior
The user is being able to upload more than 50 MB size pictures. .

### Steps to reproduce the problem
1.	Login as any user .
2.	Go to My Profile section>> Personal Information>>Avatar
3.	Upload the large size picture.  

### Screenshot/screencast
Attach or link a resource.

![image size](https://user-images.githubusercontent.com/25297859/76411422-e593f500-63b9-11ea-86e3-34ef0613cef4.png)
--

#### Environment
- OS: Windows-10
- Browser (w/version): Chrome/ Version 80.0.3987.122 (Official Build) (64-bit))
- User role (client, copilot or manager): Copilot
- Account used: topcoderconnect+copilot@gmail.com

**Comment by maxceem** [comment]
At the moment we are showing all the user avatars using Cloudfront and we resize them to the small size which we need. 

In general, I would suggest not limiting the uploading photo size, as it's easier to resize a photo using some server-side script then to force customers to do it manually. 

Do we have some server-side limitations for photos sizes or any other downsides of allowing users to upload big photos?

**Comment by vikasrohit** [comment]
@maxceem I don't think we have any limit on file upload size limit. But it might make sense to put some kind of file upload size limit e.g. 10 MB might be good for profile pic. We can pick this value from configuration to enable easy change in future.
But I think the profile pic is uploaded via member api which we don't control at present. So, I guess we have to move it to backlog for now.

**Comment by maxceem** [comment]
We can validate the size of the picture client-side if you like https://stackoverflow.com/questions/27309921/how-to-validate-the-image-size-before-uploading.

**Comment by vikasrohit** [comment]
Lets leave it for now and keep it in backlog.
